CMake/setup_KWIVER.sh.in

```diff
@@ -7,6 +7,5 @@ export VG_PLUGIN_PATH=$this_dir
 export PATH=$this_dir/bin:$PATH
 export LD_LIBRARY_PATH=$this_dir/lib:$LD_LIBRARY_PATH
 export KWIVER_PLUGIN_PATH=$this_dir/lib/modules:$KWIVER_PLUGIN_PATH
```

Comment:
Using more specific subdirectories under lib may make it more apparent what goes where. The build and install destinations can be easily changed. Putting all plugins under the lib/kwiver/{arrows,sprokit,...} would integrate better into a install tree of a larger project.